src/services/PromotionService.ts

```diff
@@ -118,17 +119,18 @@ export async function queryPromotions(filters: FilterOptions, sort?: Sort): Prom
     promotionType.forEach((promotionType: string) => promotionQueryDTO.push({ promotionType }));
   }
 
-  if (sort) {
-    const queryParams: { [paramKey: string]: string } = { sort };
-    if (sort === Sort.Distance) {
-      const {
-        coords: { latitude, longitude },
-      } = await LocationService.GeolocationPosition.getCurrentLocation();
-      queryParams.lat = `${latitude}`;
-      queryParams.lon = `${longitude}`;
-    }
-    promotionQueryDTO.push(queryParams);
-  }
+  // todo: commented out until BE PR for sort is in
```

Comment:
We can uncomment this now since https://github.com/ubclaunchpad/PromoPal-backend/pull/127 should be ready to go!

Reply:
I left this commented out actually, as there seemed to be an error in push() and I didn't want to mess things up haha

Reply:
Good point, we should merge the BE PR first before uncommenting. 

The reason why we don't need push is that I changed it from type `Record<string, string>` to type `PromotionQueryDTO`, so now we should be able to do
```
promotionQueryDTO.sort = sort
promotionQueryDTO.lat = lon
promotionQueryDTO.lon = lon
```